email-templates/src/templates/speaker-voucher-code.html

```diff
@@ -0,0 +1,186 @@
+                        </li>
+                        <li>
+                          Feel free to add t-shirts, hotel rooms or anything
+                          else you need. The voucher code will only cover the
+                          price of 1 ticket.
+                        </li>
+                        <li>Go ahead and fill in your personal information.</li>
+                        <li>
+                          In the review step, insert the following voucher code:
+                          <div
+                            class="my-5 p-10 text-center text-white bg-darker"
+                          >
+                            @{{voucherCode}}
+                          </div>
+                        </li>
+                        <li>
+                          Press "Pay with card" or "Pay with Bank Transfer" and
+                          done!
+                        </li>
+                      </ul>
+                      <div
+                        class="leading-full items-center justify-center flex mb-24"
+                      >
+                        <a
+                          href="https://pycon.it/tickets"
+                          target="_blank"
+                          class="inline-block py-16 px-24 select-none text-base font-semibold text-center no-underline text-white bg-primary hover:bg-darker"
+                        >
+                          <!--[if mso
+                            ]><i
+                              style="
+                                letter-spacing: 24px;
+                                mso-font-width: -100%;
+                                mso-text-raise: 30px;
+                              "
+                              >&#8202;</i
+                            ><!
+                          [endif]-->
+                          <span style="mso-text-raise: 16px">
+                            Open Tickets Page
+                          </span>
+                          <!--[if mso
+                            ]><i
+                              style="
+                                letter-spacing: 24px;
+                                mso-font-width: -100%;
+                              "
+                              >&#8202;</i
+                            ><!
+                          [endif]-->
+                        </a>
+                      </div>
+                      <p class="m-0 mb-8">
+                        Please try and buy your ticket as soon as possible!<br />
+                        If you have any issues, reply to this email or contact
+                        us at <a href="mailto:info@pycon.it">info@pycon.it</a>
+                      </p>
+
+                      <!-- italian -->
+                      <p class="m-0 mb-8 text-center">Versione Italiana 🇮🇹</p>
+                      <p class="m-0 mb-8">
+                        Congratulazioni per l'accettazione del tuo intervento!
```

Comment:
If you have a look at the [locale](https://github.com/pythonitalia/pycon/blob/main/frontend/src/locale/index.ts) think we almost never translate `submission` in Italian, or sometimes we call them `proposta`. `intervento` doesn't sound nice to me :)